tests/config/ncsa/ForwarderCfg.yaml

```diff
@@ -0,0 +1,57 @@
+# name of forwarder
+NAME: f99
+
+# logging directory
+LOGGING_DIR: ./
+
+# working directory where cfitio and header files are fetched/assembled
+WORK_DIR: ./
```

Comment:
Should LOGGING_DIR be writing to /var/log/iip? 

Should WORK_DIR be a configured scratch area, instead of the local directory?

Reply:
My initial thinking was this is for testing purposes and I do not want to be meddling with existing production/running software's logs by writing to the same directory. But, I could have just named test logs differently.